givitsite/coordinate/views.py

```diff
@@ -1,3 +1,67 @@
-from django.shortcuts import render
+from friendreq.models import ItemsFound, ItemRequest, STATUS_CHOICES as REQUEST_STATUS_CHOICES
+from django.http import HttpRequest, HttpResponse, Http404
+from django.shortcuts import render, redirect
+from django.contrib.auth.models import User
+from . models import CoordinatedItems
+from django.conf import settings
+from . import constants
+import sys
+import os
 
-# Create your views here.
+
+def coordinator_create_view(request):
+    if request.method == 'GET':
+        itemsToCoordinate = []
+        matchedItems = ItemsFound.objects.filter(match=True)
+        openRequests = ItemRequest.objects.filter(status=constants.request_status_open)
+        users = User.objects.all()
+
+        return render (request, 'coordinator.html', {'matchedItems':matchedItems, 'openRequests':openRequests, 'users': users})
+    else:
+        create_new_coordinations(request);
+        close_related_request(request);
+        return redirect('/coordinate')
+    
+
+def create_new_coordinations(request):
+    request_id = request.POST[constants.request_id]
+    if request.method == 'POST':
+        CoordinatedItems.objects.create(
+            friend_id = request.POST[constants.friend_id],
+            request_id = request.POST[constants.request_id], 
+            pickup_location = request.POST[constants.pickup_location],
+            drop_off_location = request.POST[constants.drop_off_location],
+            item = request.POST[constants.item],
+            date = request.POST[constants.date],
+            pickup_time = request.POST[constants.pickup_time],
+            drop_off_time = request.POST[constants.drop_off_time],
+            status = constants.item_status_coordinated
+            ) 
+
+
+def close_related_request(request):
+    ItemRequest.objects.filter(id = request.POST[constants.request_id]).update(status=constants.request_status_closed)
+
+
+def create_delivery_brief(request):  
```

Comment:
Since you're using Django already, I'd stick to using Django's data creation and transfer methods rather then invent my own (In this case use a template to generate text data instead of using low-level Python string and file manipulation). 

Why not simply use a template and generate a page? You can worry about saving it to a file later... Most web apps I've seen typically generate pages and when they want to make something appear printable or downloadable they simply do some HTTP header tricks to make the browser behave differently.

Consider that generally nobody should have manual access to the web server to get the file you're generating, also perhaps your app should not have file write access either - that can prevent a whole class of security issues.

Reply:
switched to using django forms.
dropped the brief creation feature - instead we will render coordinated items in a different view - it will be implemented in a future designated (and smaller) PR - issue #82.